mllib/src/main/scala/org/apache/spark/mllib/linalg/Vectors.scala

```diff
@@ -116,6 +116,39 @@ sealed trait Vector extends Serializable {
    *          with type `Double`.
    */
   private[spark] def foreachActive(f: (Int, Double) => Unit)
+
+  /**
+   * Number of active entries. Inactive entries are all zeros, while active entries could be zero.
```

Comment:
Better phrasing: An "active entry" is an element which is explicitly stored, regardless of its value.  Note that inactive entries have value 0.